### PR TITLE
Add Basic Carousel and ProductCard Components

### PR DESCRIPTION
**Pull request by thomHayner**
Adds some basic components and function to the Related Products area of the page.  Uses plain old React without any Redux.  The bootstrap still needs a lot of work, it is currently hardcoded for ProductCard width and height.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (thomHayner) [inline public/index.html:6]: there were a few random character at the end of this line which were rendering to the page, I simply deleted them

File: src/Components/RelatedProducts/RelatedProducts.jsx [hunk 2]
```diff
@@ -1,5 +1,14 @@
 import React from 'react';
 
-const RelatedProducts = () => (<div className="related-products">RelatedProducts PLACEHOLDER</div>);
+import Carousel from './components/Carousel/Carousel';
+
+const RelatedProducts = () => (
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RelatedProducts.jsx:5]: I replaced the PLACEHOLDER text with two carousel components, one representing the 'related-products' area, and the other as a placeholder for the 'my-outfit' area.

File: src/Components/RelatedProducts/components/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/no-access-state-in-setstate */
```
Comment (thomHayner) [inline src/Components/RelatedProducts/components/Carousel/Carousel.jsx:1]: a basic functioning carousel component which displays up to 4 ProductCards in one row and has a working clickHandler

File: src/Components/RelatedProducts/components/ProductCard/ProductCard.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (thomHayner) [inline src/Components/RelatedProducts/components/ProductCard/ProductCard.jsx:1]: a basic product card component with placeholders for all of the relevant pieces of information required by the business docs

File: public/index.html (hunk 1 above)
Comment (shawnsull3) [reply public/index.html:6]: Thanks for finding that dude! haha

**Review by shawnsull3 — APPROVED** [review]
Everything looks good to me.

**Pull request merged**